test/util/bitcoin-util-test.py

```diff
@@ -77,32 +77,36 @@ def bctest(testDir, testObj, buildenv):
     are not as expected. Error is caught by bctester() and reported.
     """
     # Get the exec names and arguments
-    execprog = buildenv["BUILDDIR"] + "/src/" + testObj['exec'] + buildenv["EXEEXT"]
+    execprog = os.path.join(buildenv["BUILDDIR"], "src", testObj["exec"]) + buildenv["EXEEXT"]
     execargs = testObj['args']
     execrun = [execprog] + execargs
 
     # Read the input data (if there is any)
     stdinCfg = None
     inputData = None
     if "input" in testObj:
-        filename = testDir + "/" + testObj['input']
+        filename = os.path.join(testDir, testObj["input"])
         inputData = open(filename).read()
         stdinCfg = subprocess.PIPE
 
     # Read the expected output data (if there is any)
     outputFn = None
     outputData = None
+    outputType = None
```

Comment:
Is this change (and the one at line 107) intentional?

Reply:
Yes, there was potential risk `outputType` can be None so I added initial outputType and exception line when outputType is None, if you think it is not needed, i can remove again

Reply:
Yes, this is a good additional check.